packages/cms-editor/src/utils/cms-utils.ts

```diff
@@ -27,3 +29,8 @@ export const isCmsReadFileDataObject = (object?: CmsDataObject): object is CmsRe
 export type CmsValueDataObject = CmsStringDataObject | CmsFileDataObject | CmsDataFileDataObject | CmsReadFileDataObject;
 export const isCmsValueDataObject = (object?: CmsDataObject): object is CmsValueDataObject =>
   isCmsStringDataObject(object) || isCmsFileDataObject(object) || isCmsDataFileDataObject(object) || isCmsReadFileDataObject(object);
+
+export const fileIcon = (fileExtension: string) => {
+  const mimeType = mime.getType(fileExtension);
+  return mimeType && mimeType.startsWith('image') ? IvyIcons.CustomImage : IvyIcons.File;
+};
```

Comment:
seems not worth to me to have a new library only to detect what icon to show here.
Ether just have a list with known extensions to show the image icon or we have already an extension -> mimetype list in the backend, so maybe we can just deliver it from there?